packages/react/src/ResetPassword.js

```diff
@@ -0,0 +1,131 @@
+import React, { PropTypes, Component } from 'react';
+import isString from 'lodash/isString';
+import { Form, FormInput } from 'react-form';
+
+class ResetPassword extends Component {
+  static propTypes = {
+    accounts: PropTypes.object,
+    Container: PropTypes.node,
+    Content: PropTypes.node,
+    Avatar: PropTypes.node,
+    ResetPasswordFields: PropTypes.node,
+    ResetPasswordPasswordField: PropTypes.node,
+    ResetPasswordPasswordConfirmField: PropTypes.node,
+    ResetPasswordButton: PropTypes.node,
+    Header: PropTypes.node,
+    Footer: PropTypes.node,
+    FormError: PropTypes.node,
+  }
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      formError: '',
+    };
+  }
+
+  onSubmit = async ({
+    password,
+  }) => {
+    const { accounts, params } = this.props;
+    this.setState({
+      formError: '',
+    });
+    try {
+      await accounts.resetPassword(params.token, password);
```

Comment:
The token will come from the router. Should it be handle like that or via a token prop directly?